assessment.md

```diff
@@ -143,39 +143,96 @@ TODO: add description for each assessment level for this skill.
 
 * Requirements engineering
 
-TODO: add description for each assessment level for this skill.
+Requirements engineering will be evaluated on this scale:
+
```

Comment:
There is no need for this statement. You can remove it and the empty space below it to remain consistent with the rest of the document. This also applies to other skills being assessed below